README.md

```diff
@@ -29,9 +29,9 @@ UnSF 1.1 (20160930)
   in other programs such as unsf.c
  * Added support for outputing files to a specific directory.
  * Fixed problems found by Coverity and Clang's static analyzer.
- * Fixed "divingly" huge total memory allocation.
- * Fixed file naming problems with foribidden characters are NTFS/FAT.
- * Support MSVC, MINGW, GCC and Clang.
+ * Fixed insanely huge total memory allocation.
```

Comment:
but but... i liked the divingly reference:
https://github.com/psi29a/unsf/issues/10#issuecomment-245404986

Reply:
That should have read 'divinely' :)